Podcast/EndpointRequest.swift

```diff
@@ -101,7 +101,7 @@ class EndpointRequest: Operation {
         return baseURLString + path
     }
     
-    func parameters() -> [String:Any] {
+    func parameters() -> [String:Any]? {
```

Comment:
what bug did it cause? this goes straight into the alamofire request initializer, so it shouldn't have any effect if its empty